app/actions/bot.ts

```diff
@@ -0,0 +1,165 @@
+import { $Values } from "utility-types";
```

Comment:
This seems strange. Is there really no built-in TypeScript way of solving whatever this does?

Reply:
I think TypeScript supports straight-forward notation if you use enums (from what I remember those don't really exist in JavaScript...) 

But the variables here (as `LOGINSTATUS`) are no enums, they're `const` that might be the reason for the cumbersome notation

Just a guess..

Reply:
Could we transform them to enums though?